src/Compilers/CSharp/Portable/Lowering/SpillSequenceSpiller.cs

```diff
@@ -839,6 +839,12 @@ public override BoundNode VisitConditionalOperator(BoundConditionalOperator node
 
         public override BoundNode VisitConversion(BoundConversion node)
         {
+            if (node.ConversionKind == ConversionKind.AnonymousFunction && node.Type.IsExpressionTree())
+            {
+                // Expression trees do not contain any code that requires spilling.
+                return node;
```

Comment:
>return node; [](start = 16, length = 12)

Is it really safe to not visit this node at all? Do we do something like that elsewhere?

Reply:
Yes, it is safe because nothing legal in an expression tree can require spilling.  This code may have to be revised if that changes.  This pass doesn't perform any other kinds of rewrites that could be applicable.  I cannot think of a precedent for this in the compiler.